.vale/fixtures/OpenShiftAsciiDoc/HeadingsAlphanumeric/testinvalid.adoc

```diff
@@ -0,0 +1,8 @@
+//vale-fixture
+= Don't Use: Special Characters in Headings
+
+//vale-fixture
+== No @Symbols! Here
+
+//vale-fixture
+== No periods.
```

Comment:
Yes but Something like this is valid and gets caught too:

== Installing the .NET plugin